src/engines.jl

```diff
@@ -32,96 +28,34 @@ function _wait_till_provisioned(engine_name, max_wait_time_s=600)
 end
 
 """
-    create_default_engine(name::String)
+    create_default_engine(name::String, size::String)
 
 Create an XS engine with default settings and the provided name.
 
 If the engine already exists, return immediately. If not, create the engine then
 return once the provisioning process is complete, or failed.
 """
-function create_default_engine(name::String)
-    size = "XS"
+function create_default_engine(name::String, size::String="XS")
```

Comment:
in case there are other attributes we want to add later, can we make `size` a kwarg instead of a positional optional arg?

Reply:
Done. The name isn't so great now, since there are no defaults. `create_engine_and_wait`, maybe?